cmd/podman/libpodruntime/runtime.go

```diff
@@ -83,6 +83,10 @@ func GetRuntimeWithStorageOpts(c *cli.Context, storageOpts *storage.StoreOptions
 	// TODO CLI flags for image config?
 	// TODO CLI flag for signature policy?
 
+	if c.GlobalIsSet("namespace") {
```

Comment:
Should we add an environment variable for this, PODMAN_NAMESPACE so users would not need to add the line all the time.

Reply:
I think we might want to put it in the config file, but an environment variable is another good call

Reply:
Well we don't have a config file yet.  (Hoping to avoid one)

Reply:
I'd prefer to configure a default namespace in the `libpod.conf` to make it a bit more convenient for configuration-management tools.

Reply:
Done. I've left it commented out by default, but uncommenting it (or adding a new line entirely, which is probably easier for config management) should work fine.